backend/src/main/java/c/team/quiz/QuizController.java

```diff
@@ -21,6 +24,7 @@
 @AllArgsConstructor
 public class QuizController {
 
+    private static final Logger LOGGER = SessionService.LOGGER;
```

Comment:
można dodać adnotacje `@Slf4j` nad klasą zamiast tego, i potem odwoływać sie do loggera `log.info("Quizzes!");`

Reply:
O dzięki, poprawię to ^^